gdal/frmts/formats_list.html

```diff
@@ -14,13 +14,15 @@
 <th>Long Format Name</th>
 <th>Code</th>
 <th>Creation</th>
+<th>Copy</th>
```

Comment:
Perhaps as a footnote we should define the terms:
Creation = Creation of a new dataset from scratch
Copy = Copy from an existing source dataset